internal/speedtest/render/render.go

```
@@ -0,0 +1,207 @@
+	if t.noColor {
+		return ""
+	}
+	return strings.Join(parts, "")
+}
+
+func (t *TTYRenderer) Render(ev Event) {
+	t.mu.Lock()
+	defer t.mu.Unlock()
+
+	if t.lastProg != "" && ev.Kind != KindProgress {
+		fmt.Fprintf(t.w, "\r%s\r", strings.Repeat(" ", len(t.lastProg)+2))
```

golangci-lint (2.11.4)

[error] Error return value of `fmt.Fprintf` is not checked

(errcheck)